projeto.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="pt-br">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Lessons Learned</title>
+    <link rel="stylesheet" type="text/css" href="XABLAU.css">
+</head>
+
+<body>
+    <header id="cabecalho">
+        <img id="logo" src="https://go.betrybe.com/hubfs/logo%20cortada%20branca.png" alt="logo-trybe">
+        <h1 id="titulo">Lessons Learned <br> Project</h1>
+        <nav id="menu">
+            <a class="link" href="#lessons">Lições</a>
+            <a class="link" href="#about">O Projeto</a>
+            <a class="link" href="#xablau">Xablau</a>
+            <a class="link" href="https://www.linkedin.com/in/rafittu/" target="_blank">LinkedIn</a>
+        </nav>
+    </header>
+    <img id="minha_foto" src="https://media-exp1.licdn.com/dms/image/C4D03AQFbSbGqLyGS_Q/profile-displayphoto-shrink_800_800/0/1554740351459?e=1619654400&v=beta&t=srFUbB-NtkW-5CQU8WvagqlDKI0YjFwp4r-ylYCQuqo" alt="foto-rafael">
+    <div id="lessons">
+        <aside class="lado-esquerdo">
+            <h2 class="subtitle">Lições</h2>
+            <h3>Aprendidas</h3>
+            <ol id="licoes_aprendidas">
+                <br>
+                <li>Linux</li>
+                <li>Unix</li>
+                <li>Shell</li>
+                <li>Bash</li>
+                <li>Git</li>
+                <li>GitHub</li>
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>Trabalho em equipe</li>
+                <li>Inteligencia emocional</li>
+                <li>COLOCAR EM ORDEM ALFABETICA!</li>
+            </ol>
+        </aside>
+        <aside class="lado-direito">
+            <h3>A aprender</h3>
+            <ul id="licoes_a_aprender">
+                <br>
+                <li>JavaScript</li>
+                <li>React</li>
+                <li>SQL</li>
+                <li>NodeJS</li>
+                <li>Arquitetura</li>
+                <li>Python</li>
+                <li>Programação orientada</li>
+                <li>Algoritimos</li>
+                <li>Estrutura de dados</li>
+                <li>Liderança</li>
+            </ul>
+        </aside>
+    </div>
+    <section>
+        <aside class="lado-esquerdo">
+            <article id="about">
+                <h2 class="subtitle">Sobre o projeto</h2>
+                <br>
+                <p>Primeiro projeto do curso de Desenvolvimento Web na Trybe! <br> O objetivo é desenvolver um site contendo as principais informações a respeito do que foi aprendido no último mês.</p>
+                <p>O site deve estar com elementos posicionados e estilizados, além de conter semântica apropriada para que seja acessível e melhor ranqueado.</p>
+            </article>
+        </aside>
+    </section>
+    <article id="box">
+        <aside class="lado-diretito">
```

Comment:
Fala @rafittu, blz?! Talvez o nome dessa classe esteja com um pequeno erro de digitação. Dei uma olhada no teu CSS e não vi uma classe correspondente lá. O que vc acha?

```suggestion
        <aside class="lado-direito">
```